src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContract.cs

```diff
@@ -1282,12 +1368,12 @@ private static bool IsTypeSerializable(Type type, HashSet<Type> previousCollecti
         {
             Type itemType;
 
-            if (type.GetTypeInfo().IsEnum ||
+            if (type.IsSerializable ||
+                type.GetTypeInfo().IsEnum ||
```

Comment:
All tests passed without the check. But I'm not sure if this is a tweak for UWP. I'd prefer to keep it here since it's not related to ISerializable.